fluentui_listitem/src/main/java/com/microsoft/fluentui/tokenized/listitem/ListItem.kt

```diff
@@ -1020,14 +1020,14 @@ object ListItem {
                 .fillMaxWidth()
```

Comment:
Is this bix needed, it is same as the Row. Maybe we can remove the Box here

Reply:
Tested it.